styles.css

```diff
@@ -87,3 +91,401 @@ body {
   height: 20px;
   margin: 10px;
 }
+
+#recent {
+  display: inline-block;
+  width: 375px;
+  height: 3000px;
+  top: 0;
+  gap: 80px;
+}
+
+#summary {
+  display: block;
+  width: 300px;
+  top: 12px;
+  word-wrap: break-word;
+}
+
+#title {
+  width: 327px;
+  height: 83px;
+  font-family: 'Crete Round', sans-serif;
+  font-weight: 400;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #1c1a19;
+}
+
+.pic_div {
+  width: 375px;
+  height: 386px;
+  display: block;
+  top: 2000px;
+}
```

Comment:
Great job @Solobarine 👏

- [ ] [Optional] Please try to display the contents of the cards with flex instead of grid. Please check out the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md).